src/Select/Select.js

```diff
@@ -62,16 +64,16 @@ const Select = React.forwardRef(({
             {...props}
             className={selectClasses}
             id={id}
-            onClick={handleClick}
+            onClick={disabled || readOnly ? null : handleClick}
             ref={ref}>
             <div className={selectControlClasses}>
                 {selected ? selected.text : placeholder}
```

Comment:
The design spec says the exclude the placeholder if it's readOnly, but fundamental-styles does not. We also don't have a min-width set (will make pr to fundamental-styles)